docs/operator-guides/faq.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_position: 3
+---
+
+# Frequently Asked Questions
+
+#### Where are the smart contract addresses used to register operator to EigenLayer?
+
+The contract addresses are located [here](https://docs.eigenlayer.xyz/operator-guides/operator-installation#goerli-smart-contract-addresses)
+
+#### Am I required to publicly host metadata url?
+Yes. You are required to host the metadata url publicly. The `metadata` url should always be available and should return a proper json response like [this](https://goerli-operator-metadata.s3.amazonaws.com/metadata.json)
+
+#### Am I required to publicly host logo in metadata json?
+Yes. You are required to host the logo publicly like [this](https://goerli-operator-metadata.s3.amazonaws.com/eigenlayer.png)
+
+#### Are there any restrictions to the logo image?
+Yes. We only support `.png` format and we strictly check the content of image. If your image doesn't satisfy the requirement then the EigenLayer App will not display the logo of your operator.
+
+#### What if I lose access to my keys?
+When you [create/import](https://docs.eigenlayer.xyz/operator-guides/operator-installation#create-and-list-keys) keys for the first time, it will ask a password to encrypt keys and once created, it will also show plaintext private key. Please make sure to backup the private key and the password. If you lose both you won't be able to get your keys back. If you lose the plaintext private key and still have your password you can your the export command to get your plaintext private key again. 
```

Comment:
Can you use relative links here `https://docs.eigenlayer.xyz/operator-guides/operator-installation#create-and-list-keys` vs `/operator-installation#create-and-list-keys`

Reply:
resolved